src/pages/search/SearchPage.tsx

```diff
@@ -19,41 +19,46 @@ const SearchPage = () => {
 
   return (
     <SyledPageContent>
-      <PageHeader>{t('common.registrations')}</PageHeader>
-      <Formik
-        initialValues={initialSearchParams}
-        onSubmit={(values) => {
-          const queryString = createSearchQuery(values);
-          params.set(SearchParam.From, '0');
-          if (queryString) {
-            params.set(SearchParam.Query, queryString);
-          } else {
-            params.delete(SearchParam.Query);
-          }
-          history.push({ search: params.toString() });
-        }}>
-        <Form>
-          <Box
-            sx={{
-              display: 'grid',
-              gridTemplateRows: 'auto auto 1fr',
-              gridTemplateColumns: { xs: '1fr', md: '2fr 5fr 2fr' },
-              gridTemplateAreas: {
-                xs: "'searchbar' 'sorting' 'filters' 'advanced' 'results'",
-                md: "'filters searchbar sorting' 'filters advanced advanced' 'filters results results'",
-              },
-              columnGap: '2rem',
-              rowGap: '1rem',
-            }}>
-            <List sx={{ gridArea: 'filters' }}>
-              <RegistrationTypeFilter />
-            </List>
-            <SearchBar />
-            <SortSelector />
-            <RegistrationSearch />
-          </Box>
-        </Form>
-      </Formik>
+      <BackgroundDiv sx={{ bgcolor: 'background.default' }}>
```

Comment:
Når jeg tenker meg om så kan vi vel faktisk bare fjerne `<BackgroundDiv>` her, til slik det var? Nå legger vi til en ny tag bare for å si at bakgrunnen skal være hvit, som den er som standard likevel
```suggestion
```